kornia/augmentation/container/augment.py

```diff
@@ -276,18 +277,23 @@ def inverse(  # type: ignore[override]
             else:
                 outputs[idx] = input
 
-        _outputs = [i for i in outputs if isinstance(i, Tensor)]
+        _outputs = [i for i in outputs if isinstance(i, Tensor) or KORNIA_CHECK_IS_LIST_OF_TENSOR(i)]
 
         if len(_outputs) == 1 and isinstance(_outputs, list):
             return _outputs[0]
```

Comment:
here can be this?
```suggestion
        return self.__packup_output__(...)
```

Reply:
It looks better if `__packup_output__ ` support this.